drop-boxes/register-omero-metadata/register-omero.py

```diff
@@ -194,10 +208,14 @@ def getPropertyMap(line, property_names):
 	property_values = line.split("\t")
 
 	for i in range(0, len(property_names)): #do not exclude first col (filename), the schema checks for it
-		##remove trailing newline, and replace space with underscore
+		## remove trailing newline, and replace space with underscore
 		name = property_names[i].rstrip('\n').replace(" ", "_")
```

Comment:
Nit: You could think about moving the rstrip and replace calls into a seperate method with documentation since it's applied multiple times and it's unclear why it's necessary here.

Reply:
Ok, I can think of making it more expressive, I will add a note here